### DISCUSSION TITLE
新增功能：常用网页显示

### DISCUSSION THREAD

**Discussion by zunlongzhou** [dukeluo/wai#88] [Ideas]
感谢作者提供这么有意思的项目哈哈哈

就是能不能把浏览器的那个最近访问常用网页列表    的功能加在旁边呢，或者这个功能做成可选的

这个功能是我日常工作生活中用的最频繁的功能了，感觉一时半会儿抛不掉hhh

**Comment by dukeluo** [comment]
我看看先 👀

**Reply by dukeluo** [reply]
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/topSites/get

**Comment by ayamir** [comment]
支持
